05-how-to-contribute-to-open-source.md

```diff
@@ -25,139 +29,292 @@ OpenSauced is a powerful tool for finding open source projects to contribute to.
+
+Below are some tips to avoid further conflicts when contributing to open source projects:
+
+#### Following Instructions
+
+Ensure you follow the instructions in the project's README or Contributing Guide, and don't miss any step.
 
-- Merge conflicts happen when two branches have made different changes to the same lines in a file. Git cannot automatically determine which change to keep, and this results in a conflict.
-- When a merge conflict occurs, Git adds special markers (<<<<<<<, =======, and >>>>>>) to the conflicted part of the file, indicating the conflicting sections from different branches.
+#### Pull Request Form
 
-### Resolving Merge Conflicts in VS Code
+Complete the template form and fill in all areas when creating a pull request.
 
-#### Prerequisites
+If a repository doesn't provide you with a pull request template, you need to have these in your pull request form:
 
-Ensure you have [VS Code](https://code.visualstudio.com/) installed on your computer and the latest version of [Git](https://git-scm.com/).
+- A descriptive title
+- A clear description of your changes
+- The [link to the related issue](https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue)
+- A screenshot or screen recording when you make a UI change
 
-#### Step-by-Step Guide
+#### Resolving Merge Conflicts Immediately
 
-1. Open VS Code: Launch Visual Studio Code and open the repository where the merge conflict exists.
-2. Check for Conflicts: If you've just performed a pull or merge operation and a conflict occurs, VS Code will automatically detect it.
-You will see a notification in the source control tab or sidebar.
-3. View Conflicted Files: Click on the conflicted file in the source control tab. VS Code will display a "Merge Conflict" section in the file.
-4. Resolve Conflicts: Inside the "Merge Conflict" section, you'll see the conflicting changes separated by the conflict markers (<<<<<<<, =======, >>>>>>). From there, decide which changes to keep or combine.
-5. Save the file: After resolving the conflicts, save the file.
-6. Stage the Resolved File: In the Source Control tab, you should see options to stage the resolved file. Click on the "Mark as Resolved" or similar button.
-7. Commit the Changes: Once all conflicts are resolved, commit your changes using VS Code's built-in source control features.
-8. Push Your Changes: After committing, push the changes to the remote repository to complete the merge process.
-9. Verify the Merge: Verify that the merge conflict has been successfully resolved by checking the remote repository or using Git tools.
+When you notice merge conflicts in your PR or if a maintainer asks you to resolve merge conflicts, fix them immediately.
```

Comment:
[suggestion] @adiati98, readers might wonder why they need to solve merge conflicts quickly, so I added an explanation that provides this information. What do you think?  
```suggestion
When you notice merge conflicts in your PR or if a maintainer asks you to resolve merge conflicts, fix them immediately. Open Source projects are always get contributions and updates, so solving them quickly keeps your branch fresh and make it easier for maintainers to review and merge your PR.